### PR TITLE
[MIG] server_environment_files_sample: Migration to 15.0

### PR DESCRIPTION
**Pull request by amh-mw**

### CODE REVIEW

File: server_environment_files_sample/__init__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (legalsylvain) [inline server_environment_files_sample/__init__.py]: Could you remove all this useless text ?

**Review by legalsylvain — APPROVED** [review]
Minor remarks. Otherwise, lgtm.

**Comment by amh-mw** [comment]
Not really sure what the pre-commit failure has to do with anything.
https://github.com/OCA/server-env/actions/runs/3640484434/jobs/6145212009#step:7:21
```
An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/git', 'fetch', 'origin', '--tags')
return code: 128
expected return code: 0
stdout: (none)
stderr:
    fatal: could not read Username for 'https://gitlab.com/': No such device or address
    
Check the log at /home/runner/.cache/pre-commit/pre-commit.log
Error: Process completed with exit code 3.
```

**Comment by amh-mw** [comment]
@OCA-git-bot Un-stale

**Comment by amh-mw** [comment]
@OCA/tools-maintainers GTG?

**Review by thomaspaulb — APPROVED** [review]

**Pull request merged**